docs/modules/ROOT/pages/type-definitions/constraints.adoc

```diff
@@ -0,0 +1,80 @@
+[[type-definitions-constraints]]
```

Comment:
I'm confused as to why this is showing as a new file - this already exists in `master`?

Reply:
mmm, I think there has been a merge issue, as this file exists with a different name. I'll remove this version as it should not have any relevant additions